app/dashboard/templates/addinterest.html

```diff
@@ -23,7 +23,20 @@
     <div class="col-12">
       <h5 class="text-center font-title">{% trans "Submit a Plan" %}</h5>
     </div>
-    {% if user_logged_in %}
+    {% if stake_required %}
+      <div class="col-12 text-center">
+        <div class="stake">
+          <div class="stake__text">
+            The bounty funder has specified that you must stake ETH to work on this bounty.  Staking ETH is a sign that you are taking the bounty seriously.  If you successfully get this bounty done, you will receive your ETH back.
```

Comment:
I for any reason I need to stop working I receive it back?

Reply:
we should figure out the triggers... i think that if you abandon the bounty then you should NOT get the stake back.